### PR TITLE
Fix: CSS folder added + minor indentation fix

### PR DESCRIPTION
**Pull request by mrsudarshanrai**
Internal style is now added as external style , fixed some html indentation and added padding 0 to body

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,97 +1,32 @@
...
-  <script src="./js/p5Vector.js"></script>
+  <script src="./script/p5Vector.js"></script>
   <!-- <script src="https://rawgit.com/anuraghazra/Candy.js/master/dist/Candy.build.js"></script> -->
   <!-- <script src="../src/lib/Candy.js"></script> -->
   <!-- <script src="./js/Candy.js"></script> -->
-  <script src="./js/p5min.js"></script>
-  <script src="./js/turtle.js"></script>
-  <script src="./js/index.js"></script>
+  <script src="./script/p5min.js"></script>
+  <script src="./script/turtle.js"></script>
```
Comment (anuraghazra) [inline index.html]: I prefer to have scripts in js folder not in scripts folder. So please change that.

File: index.html [hunk 2]
```diff
@@ -1,97 +1,32 @@
...
-      color: white;
-    }
-
-    #history p {
-      margin-top: 10px;
-      font-size: 12px;
-    }
-  </style>
+  <!--stylesheet-->
+  <link rel="stylesheet" href="./style/style.css">
```
Comment (anuraghazra) [inline index.html]: I prefer to have css files in css folder not in style folder.. So please change that.

**Review by anuraghazra — CHANGES_REQUESTED** [review]

**Comment by anuraghazra** [comment]
Looks ok! 👌

**Comment by mrsudarshanrai** [comment]
That's great ;)

**Comment by anuraghazra** [comment]
@all-contributors please add @anuraghazra for code, doc, tool, maintenance, ideas

**Comment by anuraghazra** [comment]
@all-contributors please add @anuraghazra for code, doc, tool, maintenance, ideas

**Comment by anuraghazra** [comment]
@all-contributors please add @Shudarshan for code

**Comment by anuraghazra** [comment]
@all-contributors please add @Shudarshan for code

**Pull request merged**